composer.json

```diff
@@ -1,14 +1,18 @@
 {
-	"name": "chrisboulton/php-resque",
+	"name": "spiritdead/php-resque",
```

Comment:
@danhunsaker  sorry, was for upload in the packagist for my projects for use the composer to my github is the only way for apply my fixs, this project is outdated